packages/peregrine/lib/talons/AuthModal/useAuthModal.js

```diff
@@ -2,9 +2,10 @@ import { useCallback, useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useApolloClient, useMutation } from '@apollo/client';
 
-import { useUserContext } from '../../context/user';
 import { clearCartDataFromCache } from '../../Apollo/clearCartDataFromCache';
 import { clearCustomerDataFromCache } from '../../Apollo/clearCustomerDataFromCache';
+import { useUserContext } from '../../context/user';
+import SIGN_OUT_MUTATION from '../../gql/operations/signOut.graphql';
```

Comment:
I want to get away from generic mutations/queries if possible but I also know we have some, like sign out, that are global.

I think the goal of this was to move _most_ of them next to the talons where they are used.

Reply:
Any operations I come across that have a single use I'll definitely do this. If we can come to an agreement about what to do with shared queries, I would love to get rid of this directory entirely. Simple solution to just duplicate operations would be fine by me. We can colocate fragments to a central directory, since it makes more sense to share those.

Reply:
Yea, I think duplicating operations is fine for now and we can optimize later.

Reply:
DevTime Discussion: Willing to accept duplication to delete queries out of this central location.